lingua_franca/lang/parse_en.py

```diff
@@ -635,6 +639,38 @@ def extract_duration_en(text):
     return (duration, text)
 
 
+def month_to_int_en(month):
+    if isinstance(month, int) or isinstance(month, float):
+        return int(month)
+    if isinstance(month, str):
```

Comment:
I think this should become a lang-specific dictionary, and the function should move upstairs, so that each parser can just do something like, `month_to_int(word, lang_code)`